docs/extending-documentation.md

```diff
@@ -0,0 +1,289 @@
+
+For this step we make use of a remark plugin for the MD content, similar to how you would use a babel plugin for JS content. It is called [Remark extend](https://lion-web-components.netlify.app/?path=/docs/tools-remark-extend--page).
+It will let you add, remove or replace sections or specific words.
+
+First of all we need to add the plugin to the `.storybook/main.js`:
+
+```js
+const fs = require('fs');
+const { remarkExtend } = require('remark-extend');
+
+function isLion(filePath) {
+  return filePath.indexOf('@lion/') !== -1;
+}
+
+function getLocalOverridePath(filePath, root = process.cwd()) {
+  const rel = filePath.substring(filePath.indexOf('/@lion/') + 7, filePath.length - 3);
+  return `${root}/packages/${rel}.override.md`;
+}
+
+module.exports = {
+  [...],
+  setupMdjsPlugins: (plugins, filePath) => {
+    if (!isLion(filePath)) {
+      return plugins;
+    }
+    const newPlugins = [...plugins];
+    const markdownIndex = newPlugins.findIndex(plugin => plugin.name === 'markdown');
+    const overridePaths = [`${process.cwd()}/.storybook/all.override.md`];
+    overridePaths.push(getLocalOverridePath(filePath));
+
+    let i = 0;
+    for (const overridePath of overridePaths.reverse()) {
+      if (fs.existsSync(overridePath)) {
+        const extendMd = fs.readFileSync(overridePath, 'utf8');
+        newPlugins.splice(markdownIndex, 0, {
+          name: `remarkExtend${i}`,
+          plugin: remarkExtend.bind({}),
+          options: { extendMd, filePath, overrideFilePath: overridePath },
+        });
+      }
+      i += 1;
+    }
+    return newPlugins;
+  },
+  [...],
+};
+```
+
+In the code above we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component, when needed, the `rel` needs to be the same in `lion` and your own project to be able to override the right file.
+
+In each file you need to add a specifier which section you want to override, as a first step we want to load `lea-tab` and `lea-tab-panel` also in the project:
+
+````md
+```
+::addMdAfter(':root')
+```
+
+```js script
+import '../lea-tab.js';
+import '../lea-tab-panel.js';
+```
+````
+
+And then replace each `button` or `paragraph` with them.
+
+````md
+```js ::replaceFrom(':root')
+module.exports.replaceSection = node => {
+  if (node.type === 'code' && node.value) {
+    let newCode = node.value;
+    newCode = newCode.replace(/<button slot="tab"/g, '<lea-tab slot="tab"');
+    newCode = newCode.replace(/<\/button>/g, '</lea-tab>');
+    newCode = newCode.replace(/<p slot="panel"/g, '<lea-tab-panel slot="panel"');
+    newCode = newCode.replace(/<\/p>/g, '</lea-tab-panel>');
+    node.value = newCode;
+  }
+  return node;
+};
+```
+````
+
+You can remove content:
+
+````md
+```
+::removeFrom('heading:has([value=Distribute New Elements])')
+```
+````
+
+Or you can add an extra paragraph below the content:
```

Comment:
We should explain the selectors shortly, being CSS selectors essentially, and link to `unist-util-select` or link to the remark-extend documentation README.md